File/x64/install.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
This needs to be comprehensive enough to not care about the distro. Instead of having pacman as an option for Arch, deb and rpm installs should be available options. Base it on architecture with a default of Ubuntu for apt and default of OpenSuse for rpm. Arch specifics shouldn't even be here. All changes pertaining to this script should also probably have it's own PR.